machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -47,15 +47,40 @@ It is computed using the solutions from the L0 and L1 grids with the equation
 .. math::
     f_{h=0} = f_{L0} + \frac{f_{L0}-f_{L1}}{r^{\hat{p}} - 1}
 
-Grid refinement study with ADflow
----------------------------------
-
-1. Generate fine grid (L0) with :math:`N=2^n m + 1` nodes along each edge.
-2. Coarsen the L0 grid :math:`n-1` times using :code:`cgns_utils coarsen`.
-3. Use :code:`solveCL` or :code:`solveTrimCL` in ADflow to obtain :math:`C_D` for a given :math:`C_L`.
+Grid Refinement Study
```

Comment:
Just to avoid the repetition with the main title, maybe this can be changed to `Implementation approaches` or something similar?
Also not sure if we want to keep the `Option 1..` sections on the same title level